### PR TITLE
Sets up using facebook to login to the site

### PR DESCRIPTION
**Pull request by chriskwong91**
Working facebook login.  Feature is turned off for now until other related feature (login page, etc.) is created

### CODE REVIEW

**Comment by chriskwong91** [comment]
Resolves#1

File: server/config/utils.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// utility and helper functions
+
+module.exports = {
+  isLoggedIn: (req, res, next) => {
+    if (req.isAuthenticated()) {
```
Comment (brianzhou13) [inline server/config/utils.js]: Is this function built out yet?

Comment (brianzhou13) [inline server/config/utils.js]: Unless it refers to the .isAuthenticated() definition here: https://github.com/jaredhanson/passport/blob/a892b9dc54dce34b7170ad5d73d8ccfba87f4fcf/lib/passport/http/request.js#L74

**Comment by brianzhou13** [comment]
Great work. Merging now.

**Comment by brianzhou13** [comment]
Resolve #20

**Comment by chriskwong91** [comment]
thanks for reviewing

**Pull request merged**